charts/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,319 @@
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: opentelemetry-operator-controller-manager-service-cert
+            namespace: opentelemetry-operator-system
+          type: kubernetes.io/tls
+          data:
+            tls.crt: |
+                # your signed cert
+            tls.key: |
+                # your private key
+          EOF
+        ```
+        You can also do this using `kubectl create` command.
+        ```console
+        $ kubectl create secret tls opentelemetry-operator-controller-manager-service-cert \
+            --cert=path/to/cert/file \
+            --key=path/to/key/file \
+            -n opentelemetry-operator-system
+        ```
+    - To do this after installation, you will have to install cert-manager first. Once the Operator is ready, you can override
+      the secret by putting your certificate there. What you need to do is the same as the second step above.
+
+## Add Repository
+
+```console
+$ helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+$ helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+If you didn't create the namespace `opentelemetry-operator-system` before (steps in the third method to generate the TLS cert),
+the OTEL Operator chart will be installed in the namespace automatically. Installation command example is as below.
+
+```console
+$ helm install \
+  my-opentelemetry-operator open-telemetry/opentelemetry-operator
+```
+
+However, if you create the namespace and place the TLS cert in the desired secret, you will need to set `createNamespace`
+to `false` to make sure Helm won't try to create an existing namespace, which would cause an error. Installation command example is as below.
+
+```console
+$ helm install \
+  my-opentelemetry-operator open-telemetry/opentelemetry-operator \
+  --set createNamespace=false
+```
+
+Note that `--namespace` option here won't affect on where the OTEL Operator and other resources this chart contains are installed.
+It will only affect on where the Helm chart release info is stored, which is `default` namespace by default.
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Uninstall Chart
+
+The following command uninstalls the chart whose release name is my-opentelemetry-operator.
+
+```console
+$ helm uninstall my-opentelemetry-operator
+```
+
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
+
+This will remove all the Kubernetes components associated with the chart and deletes the release.
+
+The OpenTelemetry Collector CRD created by this chart won't be removed by default and should be manually deleted:
+
+```console
+$ kubectl delete crd opentelemetrycollectors.opentelemetry.io
+```
+
+If the namespace is created by Helm instead of yourself, please skip this step. If you created the namespace and set
```

Comment:
This paragraph is a bit confusing. Perhaps simplify it? 

> If the namespace wasn't created by the Helm chart, you'll need to manually remove it as well:

Reply:
Thanks!